skops/io/_sklearn.py

```diff
@@ -169,23 +175,25 @@ def sgd_loss_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
     return state
 
 
-class SGDNode(ReduceNode):
-    def __init__(
-        self,
-        state: dict[str, Any],
-        load_context: LoadContext,
-        trusted: Optional[Sequence[str]] = None,
-    ) -> None:
-        # TODO: make sure trusted here makes sense and used.
-        self.trusted = self._get_trusted(
-            trusted, [get_module(x) + "." + x.__name__ for x in ALLOWED_SGD_LOSSES]
-        )
-        super().__init__(
-            state,
-            load_context,
-            constructor=gettype(state["__module__"], state["__class__"]),
-            trusted=self.trusted,
-        )
+if SKLEARN_SGD_LOSS:
```

Comment:
I don't think we should condition the definition of the loader class on this. The class should always be there, and it should raise a nice error when the code gets there, which can happen if the user tries to load a model which was saved with an older sklearn version.